docs/scenarios/parametric_contributing.md

```diff
@@ -0,0 +1,60 @@
+
+## Writing the tests
+
+Now that we're all setup with a working test suite and a tracer with the implemented feature, we can begin writing the new tests.
+
+First take a look at the [currently existing tests](/tests/parametric) and see if what you're trying to test is similar and can use the same methods/endpoints (in many cases this is true).
+
+For all of the exact methods already implemented you can take a look at `class APMLibrary` in the [_library_client.py](/utils/parametric/_library_client.py). If you're wondering exactly what the methods do, you can take at look at the respective endpoints they're calling in that same file in `class APMLibraryClient`.
+
+The endpoints (where the actual tracer code runs) are defined in the Http Server implementations per tracer [listed here](parametric.md#http-server-implementations). Click on the one for your language to take a look at the endpoints. In some cases you may need to just slightly modify an endpoint rather than add a new one.
+
+### If you need to add additional endpoints to test your new feature
+
+*Note:* please refer to the [architecture section](parametric.md#architecture-how-system-tests-work) if you're confused throughout this process.
+
+Then we need to do the following:
+
+* Determine what you want the endpoint to be called and what you need it to do, and add it to your tracer's http server.
+
+*Note:* If adding a new endpoint please let a Python implementer know so they can add it as well [see](parametric.md#shared-interface)
+* In [_library_client.py](/utils/parametric/_library_client.py) Add both the endpoint call in `class APMLibraryClient` and the method that invokes it in `class APMLibrary`. Use other implementations for reference.
+* Ok we now have our new method! Use it in the tests you write using the [below section](#if-the-methods-you-need-to-run-your-tests-are-already-written)
+
+### If the methods you need to run your tests are already written
+
+Make a new test file in `tests/parametric`, copying in the testing code you want to use as a base/guideline (usually the class and and one of the test methods in it).
+
+Then:
+
+* [Change the name of the feature annotation it'll fit under for the feature parity board](/docs/edit/features.md) (Not always needed e.g. `@features.datadog_headers_propagation` is used for all the propagation features)
+* Change the class and method name to fit what you're testing.
+* [Change your tracer's respective manifest.yml file](/docs/edit/manifest.md) or else the script won't know to run your new test. If you're confused at how to do this properly, search for the file you copied the test from in the manifest file and see how it's specified, you can probably copy that for your new file (make sure the path is the same).
```

Comment:
"Change" doesn't seem like the right word choice here; maybe "Choose"?

Reply:
I get your point but I think that in this case "change" makes more sense since we're asking them to copy code before. "Copy that method, ok now change the method name" Seems like more common speak and direct for us than "Copy that method, ok now choose a method name". The former would definitely be clearer to me if I were pair programming and driving.

Reply:
Feel free to reopen if you feel strongly.